File: hyeon-uk/2021/1101~1107/2920.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        char[] melody=new char[]{'c','d','e','f','g','a','b','C'};
        String asc="cdefgabC";
        String des="Cbagfedc";
        String ret="";
        Scanner scanner=new Scanner(System.in);
        for(int i=0;i<8;i++){
            int n=scanner.nextInt();
            ret+=melody[n-1];
        }
        if(ret.equals(asc)){
            System.out.println("ascending");
        }
        else if(ret.equals(des)){
            System.out.println("descending");
        }
        else{
            System.out.println("mixed");
        }

    }
}

File: jochanmin/2021/20211107/2920.cpp
#include <iostream>
#include<string>
using namespace std;
int main(void) {
	string des="87654321";
	string asc="12345678";
	string input="";
	for(int i=0;i<8;i++){
		string a;
		cin>>a;
		input+=a;
	}
	if(!des.compare(input)){
		cout<<"descending";
	}else if(!asc.compare(input)){
		cout<<"ascending";
	}else{
		cout<<"mixed";
	}
 
	return 0;
}
 